app/views/docs/realtime.phtml

```diff
@@ -121,6 +122,11 @@ let subscription = realtime.subscribe(channels: ["files"]) { response in
     <p>Using <span class="tag">Role.any()</span> on read permissions will allow any client to receive updates.</p>
 </div>
 
+<h2><a href="/docs/realtime#authentication" id="authentication">Authentication</a></h2>
+<p>Realtime authenticates using an existing user session. If you authenticate <i>after</i> creating a subscription, the subscription will not receive updates for the newly authenticated user. You will need to re-create the subscription so that it will work with the new user.</p>
```

Comment:
```suggestion
<p>Realtime authenticates using an existing user session. If you authenticate <b>after</b> creating a subscription, the subscription will not receive updates for the newly authenticated user. You will need to re-create the subscription to work with the new user.</p>
```

We don't use italics anywhere else, so let's not introduce more variance. Tiring for the eyes to adjust to.